packages-exp/auth-exp/src/core/user/user_impl.ts

```diff
@@ -196,6 +197,8 @@ export class UserImpl implements User {
     const stsTokenManager = new StsTokenManager();
     stsTokenManager.updateFromServerResponse(idTokenResponse);
 
+    assert(idTokenResponse.localId, auth.name, AuthErrorCode.INTERNAL_ERROR);
```

Comment:
The third parameter is optional and defaults to `INTERNAL_ERROR` already

Reply:
done